commit 38e0d3fd1ee170ba95f908e1606fbb8763a98593
Author: Jani Nikula <jani.nikula@intel.com>
Date:   Mon Nov 7 16:04:54 2022 +0200

    drm/i915/pxp: use <> instead of "" for headers in include/
    
    Headers in include/ should be included using the system header #include
    syntax.
    
    Fixes: 887a193b4fb1 ("drm/i915/pxp: add huc authentication and loading command")
    Cc: Tomas Winkler <tomas.winkler@intel.com>
    Cc: Vitaly Lubart <vitaly.lubart@intel.com>
    Cc: Daniele Ceraolo Spurio <daniele.ceraolospurio@intel.com>
    Cc: Alan Previn <alan.previn.teres.alexis@intel.com>
    Signed-off-by: Jani Nikula <jani.nikula@intel.com>
    Reviewed-by: Daniele Ceraolo Spurio <daniele.ceraolospurio@intel.com>
    Link: https://patchwork.freedesktop.org/patch/msgid/20221107140454.2680954-1-jani.nikula@intel.com

diff --git a/drivers/gpu/drm/i915/pxp/intel_pxp_huc.c b/drivers/gpu/drm/i915/pxp/intel_pxp_huc.c
index 7ec36d94e758..f6a3f53a1d22 100644
--- a/drivers/gpu/drm/i915/pxp/intel_pxp_huc.c
+++ b/drivers/gpu/drm/i915/pxp/intel_pxp_huc.c
@@ -3,7 +3,8 @@
  * Copyright(c) 2021-2022, Intel Corporation. All rights reserved.
  */
 
-#include "drm/i915_drm.h"
+#include <drm/i915_drm.h>
+
 #include "i915_drv.h"
 
 #include "gem/i915_gem_region.h"